testsuite/ui/views/admin/audience/developer_portal/__init__.py

```diff
@@ -2,19 +2,151 @@
 from selenium.common.exceptions import NoSuchElementException
 from wait_for import TimedOutError, wait_for
 from widgetastic.widget import GenericLocatorWidget, TextInput, Text, FileInput, Image
+from widgetastic_patternfly4 import PatternflyTable
 
 from testsuite.ui.navigation import step
 from testsuite.ui.views.admin.audience import BaseAudienceView
-from testsuite.ui.widgets import ThreescaleDropdown, DivBasedEditor
+from testsuite.ui.widgets import ThreescaleDropdown, DivBasedEditor, ThreescaleButtonGroup, ThreescaleCheckBox, \
+    CheckBoxGroup
 from testsuite.ui.widgets.buttons import ThreescaleSubmitButton, ThreescaleDeleteButton, ThreescaleCreateButton
 
 
+class DeveloperPortalContentNewPageView(BaseAudienceView):
```

Comment:
The naming of these classes is a bit off. I would opt for something like `CMSNewPageView` or `ContentNewPageView`